geonode/layers/tests.py

```diff
@@ -1021,6 +1022,14 @@ def test_batch_permissions(self):
             logger.debug(f" -- perm_spec[users] --> {perm_spec['users']}")
             self.assertTrue(user in perm_spec["users"])
 
+    # utils.py Tests
```

Comment:
Not all `utils.py` tests are under this heading (e.g. https://github.com/GeoNode/geonode/blob/546d15da321aac7220d67f00134d9d2bad492025/geonode/layers/tests.py#L538)

Reply:
I deleted the header.